cmake/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ endif()
 ###############################################################################
 
 # Boost.Circular Buffer
-find_package(Boost 1.55.0 QUIET)
```

Comment:
Can you elaborate on why this is necessary?

I am using Trial.Online on a project that requires 1.57 (although I have not built it with newer versions of Trial.Online yet.)

Reply:
boost 1.55 doesn't have the boost/core/lightweight_test_trait.hpp include, it doesn't have the core libs. I should have updated to 1.56 instead of 1.63.

Maybe we can update it to 1.57, since that's the one you're using as well? Let me know, I can change the PR accordingly.

Reply:
Either 1.56 or 1.57 is fine with me.